scripts/ninja.js

```diff
@@ -1750,7 +1752,7 @@ function main() {
     switch (subcommand) {
       case "build":
         try {
-          cp.execFileSync(vendorNinjaPath, {
+          cp.execFileSync(path.resolve(jscompDir, vendorNinjaPath), {
```

Comment:
why this change?
Can we fix vendorNinjaPath directly using `__dirname` in the def site